resources.tf

```diff
@@ -217,3 +223,48 @@ resource "aws_iam_role_policy_attachment" "emr_analytical_dataset_write_s3" {
   role       = aws_iam_role.analytical_dataset_generator.name
   policy_arn = aws_iam_policy.analytical_dataset_write_s3.arn
 }
+
+
+# Security groups
+
+resource "aws_security_group" "analytical_dataset_generation" {
+  name                   = "analytical_dataset_generation_common"
+  description            = "Contains rules for both EMR cluster master nodes and EMR cluster slave nodes"
+  revoke_rules_on_delete = true
+  #vpc_id                 = module.vpc.vpc_id
+}
+
+resource "aws_security_group_rule" "analytical_dataset_generation_egress" {
+  description       = "Allow outbound traffic from Analytical Dataset Generation EMR Cluster"
+  type              = "egress"
+  from_port         = 0
+  to_port           = 0
+  protocol          = "-1"
+  #prefix_list_ids   = [module.vpc.s3_prefix_list_id]
+  security_group_id = aws_security_group.analytical_dataset_generation.id
+}
+
+resource "aws_security_group_rule" "analytical_dataset_generation_ingress" {
+  description       = "Allow inbound traffic from Analytical Dataset Generation EMR Cluster"
+  type              = "ingress"
+  from_port         = 0
+  to_port           = 0
+  protocol          = "-1"
+  #prefix_list_ids   = [module.vpc.s3_prefix_list_id]
+  security_group_id = aws_security_group.analytical_dataset_generation.id
+}
+
+resource "aws_security_group" "analytical_dataset_generation_service" {
+  name                   = "analytical_dataset_generation_service"
+  description            = "Contains rules automatically added by the EMR service itself. See https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-man-sec-groups.html#emr-sg-elasticmapreduce-sa-private"
+  revoke_rules_on_delete = true
+  #vpc_id                 = module.vpc.vpc_id
```

Comment:
`data.terraform_remote_state.ingest.ingestion_vpc.id` is the thing you need here. Ordinarily, I'd agree to hold off on this until EMR comes along and tries to get in the same VPC. However, with ingest being on TF11 and this thing being in TF12 I'd like to get early visibility of any compatibility issues please.